### PR TITLE
Fixed FortuneAbility.

### PR DESCRIPTION
**Pull request by ArtiOJ**
Made FortuneAbility work as intended.

### CODE REVIEW

File: src/main/java/dev/corestone/lotrings/abilities/FortuneAbility.java [hunk 1]
```diff
@@ -36,17 +37,25 @@ public void switchState(RingState ringState) {
...
+        }
+
         ItemStack tool = new ItemStack(Material.DIAMOND_PICKAXE);
         tool.addUnsafeEnchantment(Enchantment.LOOT_BONUS_BLOCKS, (int) fortuneLevel);
 
-        event.setDropItems(false);
+        event.getBlock().breakNaturally(tool);
     }
-}
+
```
Comment (RTClarkV) [inline src/main/java/dev/corestone/lotrings/abilities/FortuneAbility.java:60]: does this not damage the player's tool?

**Comment by RTClarkV** [comment]
does this not damage the player's tool?

**Comment by ArtiOJ** [comment]
Fixed @RTClarkV

**Comment by ArtiOJ** [comment]
I just added all lima's changes to this, because I am bad and do not know how to push to another branch.

File: src/main/java/dev/corestone/lotrings/abilities/NecromanceAbility.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import java.util.List;
+
+public class NecromanceAbility extends AbilitySuper {
+
+    private CooldownManager cooldownManager;
+    private int summons;
+    private String summonType;
+    private double summonHP;
+    private int summonLength;
+    private HashMap<Player, List<LivingEntity>> necroSummons = new HashMap<>();
```
Comment (RTClarkV) [inline src/main/java/dev/corestone/lotrings/abilities/NecromanceAbility.java]: the hash map here is completely unnecessary. This ability and class is tied to the player's ring. You can literally do ring.getOwner() to get the UUID of the ring's owner. All you need is a list of the entities. If another ring also has this ability, it makes a new NecromancyAbility.java class for that player.

File: src/main/java/dev/corestone/lotrings/abilities/NecromanceAbility.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        }
+        cooldownManager.startCooldown();
+        System.out.println(necroSummons);
+    }
+
+    @EventHandler
+    public void onHit(EntityDamageByEntityEvent event){
+        if (event.getDamager() instanceof Player && event.getEntity() instanceof LivingEntity){
+            Player player = ((Player) event.getDamager());
+            System.out.println("Damager: "+player);
```
Comment (RTClarkV) [inline src/main/java/dev/corestone/lotrings/abilities/NecromanceAbility.java]: I think some debugging code was left here.

File: src/main/java/dev/corestone/lotrings/abilities/NecromanceAbility.java (hunk 2 above)
Comment (ArtiOJ) [reply src/main/java/dev/corestone/lotrings/abilities/NecromanceAbility.java]: Did not catch that in my code review, I will fix this when I get home.

**Comment by ArtiOJ** [comment]
Think this works, read it over.

**Pull request merged**